system/preside-objects/formbuilder/formbuilder_question_response.cfc

```diff
@@ -35,4 +35,8 @@ component displayname="Form builder: global question response" extends="preside.
 	property name="submission"   relationship="many-to-one" relatedto="formbuilder_formsubmission" required=false;
 	property name="website_user" relationship="many-to-one" relatedto="website_user"               required=false;
 	property name="admin_user"   relationship="many-to-one" relatedto="security_user"              required=false;
+
+	property name="is_website_user" type="boolean" formula="case when website_user > '' then 1 else 0 end";
+	property name="is_admin_user" type="boolean" formula="case when admin_user > '' then 1 else 0 end";
```

Comment:
Should also add `dbtype` to formula fields, in this case `dbtype="boolean"`

Reply:
dbtype added and switched to "when website_user is not null"